config/jobs/README.md

```diff
@@ -1,9 +1,213 @@
+        - --scenario=execute
+        - --
+        - make
+        - verify
+```
+
+ci-test-infra-triage: periodic, runs a container, doesn't need a repo
+
+A periodic job named "periodic-cluster-api-provider-aws-test-creds" that will
+run every 4 hours against kubernetes-sigs/cluster-api-provider-aws's master
+branch. It will run `./scripts/ci-aws-cred-test.sh` in a checkout of the repo
+located at `sigs.k8s.io/cluster-api-provider-aws`. The presets it's using will
+ensure it has aws credentials and aws ssh keys in well known locations. Its 
+logs and results are going to end up in GCS under `kubernetes-jenkins/logs/periodic-cluster-api-provider-aws-test-creds`.
+It's using the `kubekins-e2e` image which isn't recommended
+
+```yaml
+periodics:
+- name: periodic-cluster-api-provider-aws-test-creds
+  decorate: true
+  interval: 4h
+  labels:
+    preset-service-account: "true"
+    preset-aws-ssh: "true"
+    preset-aws-credential: "true"
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api-provider-aws
+    base_ref: master
+    path_alias: "sigs.k8s.io/cluster-api-provider-aws"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190618-b782256-1.15
+      command:
+      - "./scripts/ci-aws-cred-test.sh"
+```
+
+## Adding or Updating Jobs
+
+1. Find or create the prowjob config file in this directory
+    - In general jobs for `github.com/org/repo` use `org/repo/filename.yaml`
+    - For kubernetes/kubernetes we prefer `kubernetes/sig-foo/filename.yaml`
+    - Ensure `filename.yaml` is unique across the config subdir; prow uses this as a key in its configmap
+1. Ensure an [`OWNERS`](https://go.k8s.io/owners) file exists in the directory for job, and has appropriate approvers/reviewers
+1. Write or edit the job config (please see [how-to-add-new-jobs](/prow/jobs.md#how-to-configure-new-jobs))
+1. Ensure the job is configured to to display its results in [testgrid.k8s.io]
+    - The new way: add [testgrid annotations]
+    - The old way: update [`testgrid/config.yaml`]
+    - Please see the [testgrid README](/testgrid/README.md) for more details on configuation options
+1. Ensure any necessary formatting or config generation happens
+    - Run [`hack/update-config.sh`]
+    - CI will fail against your PR if this was necessary
+1. Open a PR with the changes; when it merges [@k8s-ci-robot] will deploy the changes automatically
+
+## Deleting Jobs
+
+1. Find the prowjob config file in this directory
+1. Remove the entry for your job; if that was the last job in the file, remove the file
+1. If the job had no [testgrid annotations], ensure its [`testgrid/config.yaml`] entries are gone
+1. Ensure any necessary formatting or config generation happens
+    - Run [`hack/update-config.sh`]
+    - CI will fail against your PR if this was necessary
+1. Open a PR with the changes; when it merges [@k8s-ci-robot] will deploy the changes automatically
+
+## Testing Jobs Locally
```

Comment:
Consider also linking to https://github.com/kubernetes/test-infra/blob/master/prow/build_test_update.md#how-to-test-a-prowjob

Reply:
Added a testing jobs remotely paragraph that ends up linking back there for more details